.vercelignore

```diff
@@ -6,6 +6,7 @@
 !.yarnrc
 !yarn.lock
 !package.json
+!turbo.json
```

Comment:
We use `turbo run build` in the Vercel deployment. Should we be excluding this file?